### DISCUSSION TITLE
Changing machine type to >4 cores CPU and >8GB memory

### DISCUSSION THREAD

**Discussion by OriNachum** [community/community#59867] [Codespaces] [answered]
### Select Topic Area

Question

### Body

How can I change machine type to a stronger setting?
I understand it may not be available, but there's no info about it (Is it temporary?)

I'm currently at 4CPU, 8gb mem, which is probably what most use when need more juice, and I guess I'm not alone on that.
Couldn't find a satisfying answer. (The documentation implying it's just not available is not satisfying)

Thank you!

**Comment by dmilburn** [comment, marked as answer]
Hi @OriNachum, thanks for reaching out. To request a larger machine type, please make a request to GitHub's official support team by filing a ticket here: https://support.github.com/contact?tags=rr-codespaces%2Ccat_codespaces

**Reply by AndnixSH** [reply]
Thank you, I got 8 cores and 16 cores unlocked after contacting them

**Comment by ankudinov** [comment]
@dmilburn I'd suggest to pin this issue somewhere on the top of the discussions and add some clear instructions for Codespaces users until it is correctly described in the official documentation.  
Currently the official documentation advertises capability to run various machine types up to 32-cores, 64G RAM. With a a very cryptic remark that not all machines may be available. There is no clarification explaining if the availability depends on the paid vs free plan, org membership or anything else. And there is also no documentation explaining how to upgrade this.  
Nevertheless Codespaces are GA now and I expect that community interest will grow. For ex. I'd like to publish a repository that will use 8-core machine and will be available for other github users to play. If every user will write to the support - that will quickly go out of control. Not to mention, that I've already contacted support last week and still have no response. It's clear that community accounts have no priority, but absolutely not clear if switching to Pro will improve anything in this specific case.

Codespaces is an extremely cool feature.
So, dear Github please make sizing concept easy to use and clear to understand!

**Reply by luketegel** [reply]
I agree with this, as I also got confused with this when I started to use Codespaces. I didn’t know I had to request a codespace, and the fact the documentation shows many power options and the actual version only shows 2 unless you request more is a bit misleading, especially when you just want to power up a powerful codespace and get to work quickly.
